webapp/src/components/AssetBrowse/AssetBrowse.tsx

```diff
@@ -53,8 +53,15 @@ const AssetBrowse = (props: Props) => {
     isMapViewFiltersEnabled
   } = props
 
+  const location = useLocation()
+  const history = useHistory()
   // Prevent fetching more than once while browsing
-  const [hasFetched, setHasFetched] = useState(false)
+  const latestLocation = visitedLocations[visitedLocations.length - 2]
```

Comment:
What do you think about using `last` here instead of `latest`? Latest means "the most recent one".
```suggestion
  const lastLocation = visitedLocations[visitedLocations.length - 2]
```

Reply:
done!